Detectors/TRD/workflow/src/TRDTrapSimulatorSpec.cxx

```diff
@@ -162,6 +178,7 @@ void TRDDPLTrapSimulatorTask::setOnlineGainTables()
 
 void TRDDPLTrapSimulatorTask::init(o2::framework::InitContext& ic)
 {
+  std::cout << "entering init" << std::endl;
```

Comment:
Leftover debug statement?

Reply:
that sneaky little bugger slipped through. It would only appear once, now removed.